apps/examples/testcase/le_tc/kernel/tc_sched.c

```diff
@@ -526,6 +526,65 @@ static void tc_sched_sched_getstreams(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                   :tc_sched_task_setcanceltype
+ * @brief                :This tc tests tc_sched_task_setcanceltype()
+ * @Scenario             :The task_setcanceltype() function atomically both sets the calling
+ *                        task's cancelability type to the indicated type and returns the
+ *                        previous cancelability type at the location referenced by oldtype
+ *                        If successful pthread_setcanceltype() function shall return zero;
+ *                        otherwise, an error number shall be returned to indicate the error.
+ * @API'scovered         :task_setcanceltype
+ * @Preconditions        :none
+ * @Postconditions       :none
+ * @return               :void
+ */
+#ifdef CONFIG_CANCELLATION_POINTS
+static void tc_sched_task_setcanceltype(void)
+{
+	int type;
+	int oldtype;
+	struct tcb_s *stcb;
+	int task_pid;
+
+	type = TASK_CANCEL_ASYNCHRONOUS;
+	oldtype = TASK_CANCEL_DEFERRED;
```

Comment:
oldtype is getting current set value from function. Why do you set value in here?